package desafio.processos.exception.dto;

public class ErrorDTO {
}
